lighthouse-core/audits/is-on-https.js

```diff
@@ -30,6 +30,22 @@ const UIStrings = {
     }`,
   /** Label for a column in a data table; entries in the column will be the URLs of insecure (non-HTTPS) network requests. */
   columnInsecureURL: 'Insecure URL',
+  /** Label for a column in a data table; entries in the column will be how the browser handled insecure (non-HTTPS) network requests. */
+  columnResolution: 'Resolution',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was allowed by the browser. */
+  allowed: 'Allowed',
```

Comment:
it seems like "Allowed" may be confusing (why is Lighthouse marking it as a problem if it was allowed?) but I don't have a good alternative to suggest.

Reply:
wording wise its odd that the first column is "URL" and second column is "Resolution"  
The resolution doesn't apply to that URL, but to the network request of that URL.

Adding "request" here I think helps both this and brendan's concern.

"Request allowed/blocked" ?  "Request allowed with warning"?

btw here's an example of a warning case (insecure form submission): https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/loader/mixed_content_checker.cc;l=731-747;drc=5ba77dad98cf94506cf3700f9e12fcdc65fadfb6

"Request automatically upgraded to HTTPS"

Reply:
why not change column title `Request Resolution`?

Reply:
> why not change column title `Request Resolution`?

wfm

so:

--------

`Request Resolution`

* Allowed
* Blocked
* Allowed with warning
* Automatically upgraded to HTTPS

Reply:
I still think `Request Resolution`: `'Allowed'` still raises "why is Lighthouse marking it as a problem if it was allowed?" but it seems fine to wait to see if there's any actual user confusion before coming up with something new :)

I only raise it because if it _does_ come up in the future, I don't want to see us saying "oh, well it should just be allowed and not marked as a failure, then" and instead keep following @patrickhulce's reasoning in https://github.com/GoogleChrome/lighthouse/pull/10975#issuecomment-646836765.